routes/web.php

```diff
@@ -559,7 +591,7 @@
 Route::post('/getKumpulTp/{kelas_id}/{modul_id}', 'KumpulTpController@show')->name('getKumpulTp')->middleware('loggedIn:asisten');
 
 Route::post('/createFormNilai/{praktikan_id}/{modul_id}', 'NilaiController@index')->name('createFormNilai')->middleware('loggedIn:asisten');
-Route::post('/getAllJawaban/{praktikan_id}/{modul_id}', 'NilaiController@list')->name('getAllJawaban')->middleware('loggedIn:asisten');
+Route::post('/getAllJawaban/{praktikan_id}/{modul_id}', 'NilaiController@list')->name('getAllJawaban')->middleware('loggedIn:all');
```

Comment:
yaps. like previous comment, this is ambiguous, you want to show answer, but you create the function on NilaiController, for me it's no problem, but for the next software this is not good, i hope u can change this, thankyou

Reply:
okaayyyy